core/config/engine.h

```diff
@@ -70,6 +72,8 @@ class Engine {
 
 	bool editor_hint = false;
 
+	BufferedSingletonLogger *buffered_log = 0;
```

Comment:
Shouldn't the initialized value be `nullptr` instead of `0`?

Reply:
it should indeed :)